test/components/ErrorModal.spec.tsx

```diff
@@ -0,0 +1,45 @@
+/** @jest-environment jsdom */
+import { h } from 'preact';
+import { fireEvent, render, screen } from '@testing-library/preact';
+import {
+    ErrorModal,
+} from 'js/components/ErrorModal';
+
+describe('ErrorModal', () => {
+    it('renders when there\'s an error', () => {
+        const setError = jest.fn();
+        render(
+            <ErrorModal error="My Error" setError={setError} />
+        );
+        expect(screen.queryByRole('banner')).toBeVisible();
+        expect(screen.queryByRole('banner')).toHaveTextContent('Error');
+        expect(screen.queryByText('My Error')).toBeVisible();
+    });
+
+    it('does not renders when there\'s not error', () => {
```

Comment:
"renders" -> "render"
"not error" -> "not an error" OR "no error"

Consider "there's" -> "there is" here and above.